specification/purview/Azure.Analytics.Purview.DataMap/models.tsp

```diff
@@ -0,0 +1,1963 @@
+hive_database.
+""")
+model AtlasEntityWithExtInfo {
+  ...AtlasEntityExtInfo;
+
+  @doc("An instance of an entity - like hive_table, hive_database.")
+  entity?: AtlasEntity;
+}
+
+@doc("An instance of an entity - like hive_table, hive_database.")
+model AtlasEntity {
+  ...AtlasStruct;
+
+  #suppress "@azure-tools/typespec-azure-core/bad-record-type" "Should use Record<unknown> to model Map<String,Object>"
+  @doc("Business attributes")
+  businessAttributes?: Record<unknown>;
+
+  @doc("An array of classifications.")
+  classifications?: AtlasClassification[];
+
+  @doc("The created time of the record.")
+  @encode(DateTimeKnownEncoding.unixTimestamp, int64)
+  createTime?: utcDateTime;
+
+  @doc("The user who created the record.")
+  createdBy?: string;
+
+  @doc("Custom Attribute")
+  customAttributes?: Record<string>;
+
+  @doc("The GUID of the entity.")
+  guid?: string;
+
+  @doc("The home ID of the entity.")
+  homeId?: string;
+
+  @doc("The collection ID of the entity.")
+  @visibility("read")
+  collectionId?: string;
+
+  @doc("Whether it is a shell entity")
+  isIncomplete?: boolean;
+
+  @doc("labels")
+  labels?: string[];
+
+  @doc("An array of term assignment headers indicating the meanings of the entity.")
+  meanings?: AtlasTermAssignmentHeader[];
+
+  @doc("Used to record the provenance of an instance of an entity or relationship.")
+  provenanceType?: int32;
+
+  @doc("Determines if there's a proxy.")
+  proxy?: boolean;
+
+  #suppress "@azure-tools/typespec-azure-core/bad-record-type" "Should use Record<unknown> to model Map<String,Object>"
+  @doc("The attributes of relationship.")
+  relationshipAttributes?: Record<unknown>;
+
+  @doc("""
+Status of the entity - can be active or deleted. Deleted entities are not
+removed.
+""")
+  status?: EntityStatus;
+
+  @doc("The update time of the record.")
+  @encode(DateTimeKnownEncoding.unixTimestamp, int64)
+  updateTime?: utcDateTime;
+
+  @doc("The user who updated the record.")
+  updatedBy?: string;
+
+  @doc("The version of the entity.")
+  version?: int64;
+
+  @doc("The dictionary of contacts for entities. Key could be Expert or Owner.")
+  contacts?: Record<ContactInfo[]>;
+}
+
+@doc("""
+An instance of a classification; it doesn't have an identity, this object
+exists only when associated with an entity.
+""")
+model AtlasClassification {
+  ...AtlasStruct;
+
+  @doc("The GUID of the entity.")
+  entityGuid?: string;
+
+  @doc("""
+Status of the entity - can be active or deleted. Deleted entities are not
+removed.
+""")
+  entityStatus?: EntityStatus;
+
+  @doc("Determines if propagations will be removed on entity deletion.")
+  removePropagationsOnEntityDelete?: boolean;
+
+  @doc("An array of time boundaries indicating validity periods.")
+  validityPeriods?: TimeBoundary[];
+}
+
+@doc("Captures time-boundary details")
+model TimeBoundary {
+  @doc("The end of the time boundary.")
+  endTime?: string;
```

Comment:
Are these strings or some well-known date/date-time/time formats (e.g. ISO8601)

Reply:
They are following the format as "yyyy/MM/dd HH:mm:ss"

Reply:
It seems like we do not support ISO8601 for datetime, right?
![image](https://github.com/Azure/azure-rest-api-specs/assets/78060542/61e2a3e7-df96-4ead-a292-fd98522a9938)

Reply:
ISO8601 is the default serialization for date time in json (more specifically, we recommend the [internet date/time format profile](https://datatracker.ietf.org/doc/html/rfc3339#section-5.6)). That doesn't seem to be the format you describe above, though?

Reply:
Yes. Atlas defines the format as this. And we need to be compatible with Atlas.